ui-components/index.ts

```diff
@@ -70,7 +70,7 @@ export * from "./src/helpers/preferences"
 export * from "./src/helpers/resolveObject"
 export * from "./src/helpers/useIntersect"
 export * from "./src/helpers/useMutate"
-export * from "./src/helpers/tableSummaries"
+// export * from "./src/helpers/tableSummaries"
```

Comment:
Why is this commented out, vs keeping it in or removing it?

Reply:
@emilyjablonski  Keeping it in causes a lint error. I commented it out because most of tableSummaries is commented out and I wasn't sure if we'd need it at some point:
![image](https://user-images.githubusercontent.com/60076088/164728405-1760f7cb-53d5-412d-987d-56009723c99a.png)

Should I just remove this line?

Reply:
Ahhh okay that makes sense! We may bring over the old way of doing units in the future which I think is why we left this commented out. All good 👍